src/test/java/cashleh/ParserTest.java

```diff
@@ -0,0 +1,75 @@
+package cashleh;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+
+import org.junit.jupiter.api.Test;
+
+import cashleh.commands.AddExpense;
+import cashleh.commands.AddIncome;
+import cashleh.commands.DeleteExpense;
+import cashleh.commands.DeleteIncome;
+import cashleh.commands.Exit;
+import cashleh.commands.ViewExpenses;
+import cashleh.commands.ViewIncomes;
+import cashleh.transaction.ExpenseStatement;
+import cashleh.transaction.IncomeStatement;
+import exceptions.CashLehException;
+
+public class ParserTest {
+    ExpenseStatement expenseStatement = new ExpenseStatement();
+    IncomeStatement incomeStatement = new IncomeStatement();
+    Parser parser = new Parser(new ExpenseStatement(), new IncomeStatement());
+
+    @Test
+    public void parserUnknownCommandTest() {
+        String inputString = "test";
+        try {
+            assertEquals(parser.parse(inputString), null);
+        } catch (CashLehException e) {
+            assertEquals("Aiyoh! Your input blur like sotong... Clean your input for CashLeh!", e.getMessage());
```

Comment:
I think I already commented on this line in the Parser class, but I believe we should not use slang/singlish in our code's answers (even though that requirement could also only be interpreted as being limited to actual code and not string outputs...)